### PR TITLE
Run as Non Root

### PR DESCRIPTION
**Pull request by robbo10**
Signed-off-by: Kieran Robinson <Kieran.Robinson7@gmail.com>

### CODE REVIEW

File: deploy/charts/kube-oidc-proxy/templates/deployment.yaml [hunk 1]
```diff
@@ -20,6 +20,9 @@ spec:
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
       serviceAccountName: {{ include "kube-oidc-proxy.fullname" . }}
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 65534
```
Comment (JoshVanL) [inline deploy/charts/kube-oidc-proxy/templates/deployment.yaml:25]: These should be piped through from values, rather than hardcoded here.

**Review by JoshVanL — CHANGES_REQUESTED** [review]
Cheers :) 

One comment from me.

/unassign @alljames 
/assign @robbo10

File: deploy/charts/kube-oidc-proxy/templates/deployment.yaml (hunk 1 above)
Comment (robbo10) [reply deploy/charts/kube-oidc-proxy/templates/deployment.yaml]: @JoshVanL  - How does that look ? :)

**Comment by robbo10** [comment]
Signed-off-by: Kieran Robinson Kieran.Robinson7@gmail.com

**Comment by robbo10** [comment]
@JoshVanL  - I'm unsure why that last test is failing 😕

**Pull request closed without merging**